http-server/apache2-web-server/README.md

```diff
@@ -81,7 +81,7 @@ $ mkdir -p httpd/{htdocs,logs}
 
 <br>
 
-Then, add a basic [`index.html`](https://raw.githubusercontent.com/WestleyK/examples/master/http-server/apache2-web-server/index.html) file to serve:
+Then, add a basic [`index.html`]https://raw.githubusercontent.com/sylabs/examples/master/http-server/apache2-web-server/index.html) file to serve:
```

Comment:
Add a missing `(`, it looks like this right now.. ![image](https://user-images.githubusercontent.com/2277700/48428976-762dd700-e76c-11e8-8d38-8713d4578d0c.png)

Reply:
Duh! :+1: